tatsu/_version.py

```diff
@@ -1 +1 @@
-from tatsu._config import __version__  # noqa
+__version__ = '5.7.0'
```

Comment:
This file should be `.gitignore`d

Reply:
What happens if there is no version tag on the branch?

Reply:
Error.

Reply:
I don't understand how day-to-day work can happen with that. Tags are usually applied upon a release.

Reply:
Easily. Shallow clones with insufficient depth.

Reply:
@KOLANICH, would you please post a new PR with the changes you suggest? :-)